backoffice/workflows/api/views.py

```diff
@@ -70,6 +72,52 @@ def create(self, request, *args, **kwargs):
             return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+class AuthorWorkflowViewSet(viewsets.ViewSet):
+    def create(self, request):
+
+        workflow = Workflow.objects.create(data=request.data, workflow_type="AUTHOR_CREATE")
```

Comment:
if we do it this way, I'm now thinking we don't need to pass it from inspirehep

Reply:
you mean during the airflow approve author workflow? I though it was in inspirehep that everything is stored, and that the backoffice is only for workflows

Reply:
Okay I understand, This PR can still be merge as it does not break the changes we made in inspirehep.

I will create a new PR in inspirehep to use this new api